external/fv3fit/tests/emulation/data_transform_mock.py

```diff
@@ -0,0 +1,16 @@
+from toolz.functoolz import curry
```

Comment:
It seems like these are only used in `test_data_config`. Can you move them to that model?

Reply:
Removed with the custom pipeline config removal.  I was creating the module to mock the other module which also has mostly transforms available in the namespace for `getattr`.